test/System/Sleep.test.cpp

```diff
@@ -0,0 +1,40 @@
+#include <SFML/System/Sleep.hpp>
+
+// Other 1st party headers
+#include <SFML/System/Time.hpp>
+
+#include <doctest/doctest.h>
+
+#include <chrono>
+
+using namespace std::chrono_literals;
+
+// Specialize StringMaker for std::chrono::duration specializations
+// https://github.com/doctest/doctest/blob/master/doc/markdown/stringification.md#docteststringmakert-specialisation
+namespace doctest
+{
+template <typename Rep, typename Period>
+struct StringMaker<std::chrono::duration<Rep, Period>>
+{
+    static String convert(const std::chrono::duration<Rep, Period>& duration)
+    {
+        return toString(std::chrono::nanoseconds(duration).count()) + "ns";
+    }
+};
+} // namespace doctest
```

Comment:
This appears redundant with something similar in Time.test.cpp but it's subtly differently. It has to print nanoseconds instead because that's the difference type between two `std::chrono::steady_clock::time_point`s.